app/views/notes/index.html.erb

```diff
@@ -1,58 +1,30 @@
   <% elsif params[:action] == "author" %>
-    <h3><%= raw translation('notes.index.research_by', :url1 => "/people/"+@user.name, :user_name => @user.name) %> <small><%= raw translation('notes.index.research_notes_and_wiki_edits', :count1 => @user.note_count, :word1 => 'note'.pluralize(@user.note_count), :count2 => @user.revisions.count, :word2 => 'edit'.pluralize(@user.revisions.count)) %> <% if @user.notes.size > 0 %><%= raw translation('notes.index.starting_time', :time => time_ago_in_words(@user.created_at)) %><% end %></small></h3>
+  <h3><%= raw translation('notes.index.research_by', :url1 => "/people/"+@user.name, :user_name => @user.name) %> <small><%= raw translation('notes.index.research_notes_and_wiki_edits', :count1 => @user.note_count, :word1 => 'note'.pluralize(@user.note_count), :count2 => @user.revisions.count, :word2 => 'edit'.pluralize(@user.revisions.count)) %> <% if @user.notes.size > 0 %><%= raw translation('notes.index.starting_time', :time => time_ago_in_words(@user.created_at)) %><% end %></small></h3>
   <% else %>
-    <h2><%= raw translation('notes.index.research_notes_ideas_and_documentation') %></h2>
+  <h2><%= raw translation('notes.index.research_notes_ideas_and_documentation') %></h2>
   <% end %>
-  <br/>
+  <br />
   <% if params[:action] == "liked" %><a style="margin-right:9px; margin-top: -33px;" class="float-right btn btn-outline-secondary btn-sm" href="/feed/liked.rss"><i class="fa fa-rss"></i></a><% end %>
 
-    <ul class="nav nav-pills notes-pills">
-      <li class="nav-item">
-        <a class="nav-link <% if params[:controller] == "notes" %> active <% end %>" href="/notes/"> 
-          <i class="fa fa-file"></i>
-          <span class="d-lg-inline">
-            <span class="d-none d-md-inline"> <%= translation('notes.index.research_notes') %></span>
-          </span>
-        </a>
-      </li>
-      <li class="nav-item">
-        <a class="nav-link" href="/questions/">
-         <i class="fa fa-question-circle"></i>
-         <span class="d-lg-inline">
-          <span class="d-none d-md-inline"> <%= translation('notes.index.questions') %></span>
-         </span>
-        </a>
-      </li>
-      <li class="nav-item">
-        <a class="nav-link" href="/wiki/">
-        <i class="fa fa-book"></i>
-        <span class="d-lg-inline">
-          <span class="d-none d-md-inline"> <%= translation('notes.index.wiki_pages') %></span>
-        </span>
-        </a>
-      </li>
-       <li class="nav-item">
-        <a class="nav-link" href="/comments/">
-        <i class="fa fa-comment"></i>
-        <span class="d-lg-inline">
-          <span class="d-none d-md-inline"> <%= translation('notes.index.comments') %></span>
-        </span>
-        </a>
-      </li>
-    </ul>
-    <%= render :partial => "notes/sidebar" %>
+  <% if params[:action] == "author" %>
```

Comment:
Nice, i like this conditional switch. CC @imajit that this could interact with some of your translation code but should be OK.